### ISSUE TITLE
Use a Heroku service hook on Github for deployment

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/thimble.mozilla.org#1699]
Let's use the Github Integration feature of Heroku so that the deployment to staging will not happen via Travis.
To do this, we need to create an`app.json` file that will hold the config for setting up the app on Heroku.
I don't think we will have the same problem that we did with `learning.moz.org` for environment variables because any environment variables that we use during the build are (I believe) the same on staging and production.

The main reason I want to do this is more of a means to an end (since our current Travis based deployment is fine). Switching to Github integration will allow us to create review apps that will let us test PRs online without having to go through the whole process of testing PRs locally(fetching the branch, setting vagrant up, etc.).

@cadecairos does this sound sane? (I also might need you to review it when I'n done)

**Comment by Pomax** [comment]
ping @cadecairos just to make sure this wasn't lost between the cushions somewhere

**Comment by gideonthomas** [comment]
Nope, I talked to him about it. This might be a pretty big task to do because of the different dependencies (mainly OAuth being the problem). So, I think we might de-prioritize this for now (because the cost of work to be done is much more than the benefit we get out of it).

**Comment by Pomax** [comment]
that works too.
